### PR TITLE
Adjusted link message to be above hotbar

### LINKED ISSUE #1251: [Suggestion] Put linking message to the area for "Press LShift to dismount"
This may make things neater. Will decide to implement or not per @CovertJaguar's opinion,

### PR DESCRIPTION
**Pull request by liach**
Closes #1251

### CODE REVIEW

**Comment by liach** [comment]
Should I change outfitted track placement message to be like these too?

**Comment by Generalcamo** [comment]
This adds basically consistency with the changes in 1.11. I approve.

My opinion is that everything that isn't a chat command should appear in the hotbar.

**Comment by liach** [comment]
So I think we can put the message for track placement and goggles auras here too, but not those for commands.

Moreover, I will leave those more informatic ones as it is, such as messages from magnifying glass. Track placement, cart linking, and goggles auras are easy to spam logs. (You can see goggles aura in item tooltips too)

File: src/main/java/mods/railcraft/common/plugins/forge/ChatPlugin.java [hunk 1]
```diff
@@ -65,6 +67,21 @@ public static void sendLocalizedChatFromServer(@Nullable EntityPlayer player, St
...
+        if (player instanceof EntityPlayerMP && Game.isHost(player.worldObj)) {
+            EntityPlayerMP target = (EntityPlayerMP) player;
+            for (int i = 0; i < args.length; i++) {
+                if (args[i] instanceof String) {
+                    args[i] = translateMessage((String) args[i]);
+                } else if (args[i] instanceof GameProfile) {
+                    String username = ((GameProfile) args[i]).getName();
+                    args[i] = username != null ? username : "[unknown]";
+                }
+            }
```
Comment (CovertJaguar) [inline src/main/java/mods/railcraft/common/plugins/forge/ChatPlugin.java]: Copy paste code block? Split it out into a function please.

**Review by CovertJaguar — CHANGES_REQUESTED** [review]

**Review by CovertJaguar — APPROVED** [review]

**Pull request merged**